controller/registries/nacos/config.go

```diff
@@ -289,15 +289,15 @@ func (reg *Nacos) Reload(c registrytype.RegistryConfig) error {
 
 	reg.version = config.Version
 
+	reg.lock.Lock()
+	defer reg.lock.Unlock()
+
```

Comment:
Why change the scope of the lock?

Reply:
Because it could cause data race

Reply:
@lyt122 
Got it. Thanks for your explanation.